product_docs/docs/migration_toolkit/55/05_installing_mtk/index.mdx

```diff
@@ -26,10 +25,3 @@ You can install Migration Toolkit on:
 - [Mac OS X](install_on_mac)
```

Comment:
The linux and Windows installation pages include the following statement at the end of the page:

> After installing Migration Toolkit with Stack Builder, you must install the appropriate source-specific drivers before performing a migration. See [Installing Source-Specific Drivers](https://deploy-preview-3176--edb-docs-staging.netlify.app/docs/migration_toolkit/latest/05_installing_mtk/#installing_drivers) for more information.

However, it does not appear to be included in the Mac installation page.  Is this intentional?